account_asset_management_xlsx/report/report_account_asset_xlsx.py

```diff
@@ -0,0 +1,459 @@
+            "value_residual": "=SUM({})",
+            "salvage_value": "=SUM({})",
+            "amo_ant": "=SUM({})",
+            "amo_de_lan": "=SUM({})",
+            "cum_amo": "=SUM({})",
+            "val_nette": "=SUM({})",
+        }
+        cell_format = resources["fm_table_header_dark_grey"]
+        table_infos = self._get_table_infos(resources)
+        for col_index, column in enumerate(table_infos, 0):
+            col_pos = col_index
+            cell_value = summary_data.get(column, "")
+            if "=SUM" in cell_value:
+                cell_format = resources["fm_table_number_bold_dark_grey"]
+                cell_formula_value = cell_value.format(
+                    ", ".join(summary_column_info[column])
+                )
+                sheet.write_formula(row_pos, col_pos, cell_formula_value, cell_format)
+            else:
+                sheet.write(row_pos, col_pos, cell_value, cell_format)
+        resources.update(
+            {
+                "row_pos": row_pos + 1,
+                "summary_column_info": summary_column_info,
+            }
+        )
+        return resources
+
+    def _get_company_logo_path(self):
+        company = self.env.user.company_id
+        company_logo = company.logo
+        company_logo_path = get_record_image_path(
+            record=company,
+            image=company_logo,
+        )
+        return company_logo_path
+
+    def _get_created_by_info(self, wizard_rec):
+        today = fields.Date.from_string(fields.Date.today())
+        formated_today_str = today.strftime("%d/%m/%Y")
+        created_by_header = "Créé le : "
+        created_uid = wizard_rec._context.get("uid", self.env.uid)
+        created_user = self.env["res.users"].browse(created_uid)
+        created_by_info = f"{formated_today_str} par {created_user.name}"
+        return created_by_header, created_by_info
+
+    def _get_summary_column_info(self):
+        return {
+            "value": [],
+            "value_residual": [],
+            "salvage_value": [],
+            "amo_ant": [],
+            "amo_de_lan": [],
+            "cum_amo": [],
+            "val_nette": [],
+        }
+
+    def _get_table_infos(self, resources):
+        return {
+            "name": {"str": "Nom de l'immobilisation"},
+            "state": {"str": "Statut"},
+            "date": {"str": "Date", "format": resources["fm_table_date"]},
+            "value": {
+                "str": "Valeur brute",
+                "type": "formula",
+                "format": resources["fm_table_number"],
+            },
+            "salvage_value": {
+                "str": "N/amortissable",
+                "type": "formula",
+                "format": resources["fm_table_number"],
+            },
+            "method": {"str": "Méthode"},
+            "method_number": {"str": "Nb. amort."},
+            "prorata": {"str": "Prorata"},
+            "amo_ant": {
+                "str": "Amo. ant.",
```

Comment:
```suggestion
                "str": "Beg. Acc. Dep.",
```
it's Beginning Accumulated Depreciation